doc/modules/classes.rst

```diff
@@ -68,6 +68,7 @@ Functions
    base.clone
    base.is_classifier
    base.is_regressor
+   base.is_clusterer
```

Comment:
Actually, now that we switch website, you need to edit the `doc/api_reference.py`. You can search for `is_classifier` in the file and add the `is_clusterer` just below.

Reply:
Okay, done :-)